docker/local.env

```diff
@@ -0,0 +1,4 @@
+export TAG=$(git rev-parse --abbrev-ref HEAD)
+export SERVER_HOST=localhost
+export SERVER_PORT=9000
+export SECRET_KEY="u?:IPZT:Cun;SP>edigHDs:cpUf2cY\`<7D>b9lHg98^NsVGn]:eLCIjz1Tt_VyIv"
```

Comment:
TODO: how was this key generated? Add a comment.

Reply:
Command: `sbt playGenerateSecret | grep -i 'generated' | cut -d ' ' -f5`